solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreakerManager.java

```diff
@@ -38,27 +35,11 @@
  * <p>NOTE: The current way of registering new default circuit breakers is minimal and not a long
  * term solution. There will be a follow up with a SIP for a schema API design.
  */
-public class CircuitBreakerManager implements PluginInfoInitialized {
-  // Class private to potentially allow "family" of circuit breakers to be enabled or disabled
-  private final boolean enableCircuitBreakerManager;
+public class CircuitBreakerManager {
```

Comment:
Wondering if the last two paragraphs of the class-level javadoc can be omitted or replaced now then and perhaps the class could even be marked final to make it clear that extending it is no longer intendeds?
```suggestion
public final class CircuitBreakerManager {
```